doc/api/_toc.md

```diff
@@ -53,6 +53,7 @@
 * [Utilities](util.html)
 * [V8](v8.html)
 * [VM](vm.html)
+* [Worker](worker.html)
```

Comment:
nit: perhaps rename to `[Worker Threads](worker_threads.html)` to align with the module rename?

Reply:
@jasnell Yup, done!